test/runtests.jl

```diff
@@ -1895,6 +1982,7 @@ else  # run HiGHS tests
                     end
                 end 
                 
+                # TODO: refresh tests and test values as needed after Cambium update; revised values are in test_with_xpress.jl
```

Comment:
test_with_xpress.jl is using a different/updated Cambium version/data?

Reply:
@Bill-Becker These are the emissions tests that Alex commented out a while back because they weren't compatible with the HiGHS solver. The test values are slightly updated because of the AVERT data update that we made in this PR. Alex added this note to indicate that once we are able to run these tests with a non-Xpress solver, we should update the values here. Does that all make sense?